code/modules/paperwork/photocopier.dm

```diff
@@ -317,6 +318,7 @@
 		for(var/i in copies to 1 step -1)
 			if(!copyass())
 				break
+			sleep(PHOTOCOPIER_DELAY)
```

Comment:
Why not just make the sleep() that's already in the copyass() proc apply for both human and non-human mobs, and do away with the one here? It looks like the logic would work as-is, but it might be a little less roundabout if you changed it.

Reply:
Currently requesting approval to see if I can do a more major refactor on this